products/bash/release-apic.sh

```diff
@@ -53,6 +56,10 @@ spec:
     accept: true
     use: nonproduction
   profile: n3xc4.m16
+  gateway:
+    openTracing:
+      enabled: ${tracing}
```

Comment:
Does this work when `-t` not specified? Should `tracing` be set to `false` by default so a value is passed here?